dotnet-statsig-tests/Client/ClientOptionsTest.cs

```diff
@@ -0,0 +1,85 @@
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Statsig;
+using Statsig.Client;
+using WireMock;
+using WireMock.RequestBuilders;
+using WireMock.ResponseBuilders;
+using WireMock.ResponseProviders;
+using WireMock.Server;
+using WireMock.Settings;
+using Xunit;
+
+namespace dotnet_statsig_tests
+{
+    [Collection("Statsig Singleton Tests")]
```

Comment:
This is so test the use the Statsig singleton do not class. I don't think you are using the Statsig singleton, so you can remove this.